hackclub.com.yaml

```diff
@@ -1329,6 +1332,9 @@ psn:
   type: CNAME
```

Comment:
You'll have to remove this one to make it work!

Reply:
where would it go then? i need this for the site to work w/ vercel

Reply:
these are the netlify ones aren't they?

Reply:
no these are for vercel?